src/Payum/Stripe/StripeCheckoutGatewayFactory.php

```diff
@@ -16,7 +16,7 @@ class StripeCheckoutGatewayFactory extends GatewayFactory
      */
     protected function populateConfig(ArrayObject $config)
     {
-        if (!class_exists('Stripe')) {
+        if (!class_exists('\Stripe\Stripe')) {
             throw new \LogicException('You must install "stripe/stripe-php" library.');
```

Comment:
I am thinking of adding a version here. So it is clear what version of the library user has to install. wdyt?